frontend/app.vue

```diff
@@ -1,6 +1,9 @@
 <template>
     <NuxtLayout>
-        <Toaster position="bottom-right" rich-colors />
+        <Toaster position="bottom-right" />
         <NuxtPage />
+        <DevOnly>
+            <DevTools />
+        </DevOnly>
```

Comment:
Should this be here?

Reply:
This is just for development, it will not be there in prod